### PR TITLE
Add avatar and badge component

### PR DESCRIPTION
**Pull request by sumeetyeola**
- Screenshots
Screenshot (209)
Screenshot (211)
Screenshot (214)
Screenshot (215)
- Not yet responsive. Will try at end.

### CODE REVIEW

File: components/Badge/badge.css [hunk 1]
```diff
@@ -0,0 +1,62 @@
+.badge-avatar {
+  position: relative;
+  width: fit-content;
+}
+.badge-avatar img {
```
Comment (ruchiragoswami) [inline components/Badge/badge.css:5]: For readability, one empty line is suggested at the end of each code block

**Review by ruchiragoswami — APPROVED** [review]
Good Work Sumeet! 
Just one suggestion I have, for readability, one empty line is nice to have at the end of each code block in CSS file :)

File: components/Badge/badge.css (hunk 1 above)
Comment (sumeetyeola) [reply components/Badge/badge.css:5]: Thanks Ruchi for pointing out. I'll keep that in mind.

**Review by VishalPatil18 — APPROVED** [review]
The code looks quite clean and works well. Great work @sumeetyeola 👍

**Comment by sumeetyeola** [comment]
> The code looks quite clean and works well. Great work @sumeetyeola 👍

Thankyou @VishalPatil18

**Pull request merged**